include/tilck/kernel/syscalls.h

```diff
@@ -134,7 +134,8 @@ CREATE_STUB_SYSCALL_IMPL(sys_sigpending)
 CREATE_STUB_SYSCALL_IMPL(sys_sethostname)
 CREATE_STUB_SYSCALL_IMPL(sys_setrlimit)
 CREATE_STUB_SYSCALL_IMPL(sys_old_getrlimit)
-CREATE_STUB_SYSCALL_IMPL(sys_getrusage)
+
+int sys_getrusage(int who, struct k_rusage *user_buf);
```

Comment:
STYLE: can you move this close to the function below?

Reply:
done!